tests/optimize/test_backtesting.py

```diff
@@ -648,7 +648,7 @@ def test_backtest_one(default_conf, fee, mocker, testdatadir) -> None:
     processed = backtesting.strategy.advise_all_indicators(data)
     min_date, max_date = get_timerange(processed)
     result = backtesting.backtest(
-        processed=processed,
+        processed=dict(processed),
```

Comment:
i think `advise_all_indicators()` will always return a dict ... 

this makes me curious what error/problem you ran into that made you explicitly cast it to dict?

Reply:
This is the ugly wart of this change. `backtest()` modifies `processed` dict so i had to make a copy to not break the test.

Reply:
that's not necessarily a problem - but i think using `deepcopy` for that would clarify the intend.

dict will serve the same purpose - but it's completely unclear that you're doing it in order to get a copy of the dict.